metrics/grafana/client_java_summary.json

```diff
@@ -174,7 +197,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(client_java_raw_requests_latency_sum[1m])) by (type) / sum(rate(client_java_raw_requests_latency_count[1m])) by (type)",
+          "expr": "sum(rate(client_java_raw_requests_failure_total[1m])) by (type)",
```

Comment:
how about we change the left-axis scale of all graphs containing `xxx latency` from `linear` to `log (base 10)`?

Reply:
Just out of curiosity, why do we need the log scale instead of linear? Linear may more is intuitive to me.